test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -241,6 +241,52 @@ function prepare-log-file {
 	chown root:root $1
 }
 
+# A helper function for removing salt configuration and comments from a file.
+# This is mainly for preparing a manifest file.
+#
+# $1: Full path of the file to manipulate
+function remove-salt-config-comments {
+	# Remove salt configuration.
+	sed -i "/^[ |\t]*{[#|%]/d" $1
+	# Remove comments.
+	sed -i "/^[ |\t]*#/d" $1
+}
+
+# Prepare the abac policy file from the salt template.
+function setup-abac-authz-policy-file {
```

Comment:
Thanks for the pointers @wojtek-t @deads2k . After some tweaking with the rbac bindings in e2e, I've narrowed down the access rights for kubemark to the optimal (both necessary and sufficient), pretty much what the comments in the e2e-rbac-bindings yaml files were asking to do. For example, the admin-binding was not required in kubemark as currently I guess we do not have a usecase for 'view' access to any external entity (outside kubecfg, kubelet, proxy and controller-manager).